--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -543,6 +543,9 @@

 set_source_files_properties(resources/qtquickcontrols2.conf PROPERTIES QT_RESOURCE_ALIAS qtquickcontrols2.conf)
 set_source_files_properties(${SDL_GAMECONTROLLERDB_PATH} PROPERTIES QT_RESOURCE_ALIAS gamecontrollerdb.txt)
+if(QGC_GSTREAMER_SHADER_RESOURCES)
+    set_source_files_properties(${QGC_GSTREAMER_SHADER_RESOURCES} PROPERTIES QT_RESOURCE_ALIAS org/freedesktop/gstreamer/qml6/RGBA.frag.qsb.external)
+endif()
 qt_add_resources(${CMAKE_PROJECT_NAME} "qgcresources_cmake"
     PREFIX "/"
     FILES
